### ISSUE TITLE
Bower integration

### ISSUE THREAD

**Issue by command-tab** [bennadel/httpi#1]
Would like to see the addition of a bower.json file so httpi could be quickly added to a project via Bower.

**Comment by bennadel** [comment]
@commandtab To be completely transparent, I am not too familiar with Bower. One of my co-workers gave me an overview of it last week and it seems like an "NPM for the Browser" kind of package manager. So, I assume bower.json is like package.json. I don't see why we can't do something like that :)

**Comment by egor-smirnov** [comment]
@bennadel just filed pull request in order to help with bower integration.

**Comment by aljopro** [comment]
@bennadel I agree that you should pull this in to your repository. You will find many angular developers more eager to use your library with this functionality.

**Comment by mzoellner** [comment]
Nice little library. 
+1 for bower integration.
@bennadel Please pull it in
